app/views/posts/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>show all posts by a spesific user</h1>
\ No newline at end of file
+<div class="all-post-container">
+    <%= render partial: 'users/user', locals: { user: @user } %>
+    <div class="posts-container">
+        <% @posts.each do |post|%>
+            <div class="post-comments-container">
+                <%= render partial: 'posts/post', locals: { post: post } %>
+                <div class="comments-container">
+                    <% post.five_recent_comments_post.each do |comment| %>
+                        <p class="post-comment"><%= @user.name %>: <%= comment.text %></p>
+                    <%end%>
+                </div>
+            </div>
+        <%end%>
+    </div>
+    <button class="see-all-post-btn">Pagination</button>
+</div>
```

Comment:
- [ ] Kindly position the elements to fit the wireframe.

  Your Implementation | Wireframe
  :---:|:---:
  ![image](https://github.com/nilab-khwaja/blog-app/assets/54848363/45aee435-422f-4e5b-957d-b1abbba0aa4d) | ![image](https://github.com/nilab-khwaja/blog-app/assets/54848363/5f9f24cb-baab-4ad9-bc31-cf37d80b9f61)